File: src/main/java/com/chatsummary/bot/repository/ChatConfigRepository.java
package com.chatsummary.bot.repository;

import com.chatsummary.bot.model.ChatConfig;
import java.util.Optional;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChatConfigRepository extends MongoRepository<ChatConfig, String> {

    Optional<ChatConfig> findByChatId(long chatId);
}

File: src/main/kotlin/com/chatsummary/bot/repository/ChatConfigRepository.kt
package com.chatsummary.bot.repository

import com.chatsummary.bot.model.ChatConfig
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface ChatConfigRepository : MongoRepository<ChatConfig, String> {
    fun findByChatId(chatId: Long): ChatConfig?
}
